.github/ISSUE_TEMPLATE/example-request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Example request
+about: Suggest an use case example you want to see
+title: I want to see ...
```

Comment:
Should this use a similar pattern to [Content Improvement]?

Reply:
That's right. Adding it now